### PR TITLE
Register function

### PR DESCRIPTION
**Pull request by SChaudhary99**
I am creating a new PR for registration and login feature as pervious version had some issues such as TYPOs and data handling and exception handling. 

In the previous version we needed to insert roles as USER and ADMIN manually. but in current version i have resolved that issue with CommandLineRunner.

also I had few Typos which needed to be resolved.
Screenshot 2023-06-12 at 5 03 16 AM
Screenshot 2023-06-12 at 5 03 33 AM
Screenshot 2023-06-12 at 5 03 50 AM
Screenshot 2023-06-12 at 5 03 09 AM

### CODE REVIEW

**Comment by Surinder07** [comment]
I dont think so you need to display passwords in any form in the UI, no user should have access to that.

**Comment by SChaudhary99** [comment]
Yes sir, I have removed the password row, I added that to see if the password is encrypting.

File: src/main/java/com/example/job_service/config/SimpleUrlAuthenticationSuccessHandler.java
Comment (nitika807) [inline src/main/java/com/example/job_service/config/SimpleUrlAuthenticationSuccessHandler.java:1]: This code is yet to be reviewed.

**Review by nitika807 — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -21,12 +21,42 @@
...
+            <groupId>com.h2database</groupId>
+            <artifactId>h2</artifactId>
             <scope>runtime</scope>
+        </dependency>-->
+
+        <dependency>
+            <groupId>org.springframework.security</groupId>
+            <artifactId>spring-security-test</artifactId>
         </dependency>
+
```
Comment (rohanbeldar) [inline pom.xml:59]: looks good

**Review by rohanbeldar — APPROVED** [review]

**Review by navtej-singh — APPROVED** [review]
Everything looks good!

**Review by Surinder07 — APPROVED** [review]

**Pull request merged**